docs/hugo/content/guide/aso-controller-settings-options.md

```diff
@@ -165,6 +189,22 @@ Note that the resource manager audience is referred to as "activeDirectoryResour
 
 **Required**: False
 
+**[Allowed scopes]( {{< relref "authentication#credential-scope" >}} )**: Global, namespace, or per-resource
+
+### AZURE_ADDITIONAL_TENANTS
+
+The list of (comma-separated) additional tenants the operator can authenticate with.
+This is required when performing cross-tenant authentication. See the
+[Entra documentation](https://learn.microsoft.com/entra/external-id/cross-tenant-access-overview) for more details.
+
+**Format:** `string` (comma-separated tenant GUIDs)
```

Comment:
Do the GUIds need to be jammed up together (`foo,bar,baz`) or can you have spaces to make things easier to read (`foo, bar, baz`)?

Reply:
Spaces are allowed.

Reply:
I updated the docs to clarify this.